packages/dev/core/src/scene.ts

```diff
@@ -4975,7 +4979,6 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
     ): Nullable<PickingInfo> {
         // Dummy info if picking as not been imported
         const pi = new PickingInfo();
-        pi._pickingUnavailable = true;
         return pi;

```

Comment:
I forgot if we determined if dummy pickingInfo was necessary? If so, then maybe:
```javascript
return new PickingInfo();
```

Same below.